### PR TITLE
Clean up README

### PR DESCRIPTION
**Pull request by cyrneko**
Description from the commit message:
```
This does some changes to the README to make it get rendered correctly, with linebreaks and lists.
- Add explicit line-breaks (<br>) where necessary to reflect the raw markdown
- turn l7-10 into a proper bulleted list
- switch out table-based codeblocks with proper codeblocks
```

Also, I synced with upstream right before this PR, so there is one dangling merge commit that can be ignored.

### CODE REVIEW

**Comment by teamcons** [comment]
Yeah sorry, initial one was very quick and dirty

File: README.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
 
 # eOSdock-themeswitcher
 
-To use with eOS new dock
+To use with eOS new dock<br>
```
Comment (RedNeath) [inline README.md:4]: You can use two whitespaces (just like `  `) at end of line for a line break (better to use native markdown than html but it's fine in the end) ^^'

File: README.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
-Plank-Classic: the old Plank look
-Princess Eyebleed : Something horribly pastel
-Aqua : Something more MacOS-y
+- Stellas Special: My own take. Close to default
+- Plank-Classic: the old Plank look
+- Princess Eyebleed : Something horribly pastel
+- Aqua : Something more MacOS-y
 
-THE SCRIPT WORKS; BUT EVERYTHING IN THIS REPO IS BRAND NEW AND THERES VERY LITTLE YET
+THE SCRIPT WORKS; BUT EVERYTHING IN THIS REPO IS BRAND NEW AND THERES VERY LITTLE YET<br>
```
Comment (RedNeath) [inline README.md:12]: Same here

File: README.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
 
 login-logout for change to take effect, or do
-<table><tr><td>$ killall io.elementary.dock</td></tr></table>
-
+```bash
+$ killall io.elementary.dock</td></tr></table>
+```
 
-The file doesnt overwrite gtk.css, and reverts to default without leaving anything else.
+The file doesnt overwrite gtk.css, and reverts to default without leaving anything else.<br>
```
Comment (RedNeath) [inline README.md:33]: Same ^^

**Comment by RedNeath** [comment]
Woops too late visibly ^^'

**Pull request merged**